scapy/modules/krack/crypto.py

```diff
@@ -144,7 +146,7 @@ def gen_TKIP_RC4_key(TSC, TA, TK):
     assert len(TSC) == 6
     assert len(TA) == 6
     assert len(TK) == 16
-    assert all(isinstance(x, (int, long)) for x in TSC + TA + TK)
+    assert all(isinstance(x, integer_types) for x in TSC + TA + TK)
```

Comment:
For this, we have used every where else in scapy a static call:
ˋimport scapy.modules.six as six`

Then directly
`six.integer_types`

It’s not needed but would be nice to keep code consistency